Source/Core/VideoCommon/OnScreenUI.cpp

```diff
@@ -343,6 +342,8 @@ void OnScreenUI::SetScale(float backbuffer_scale)
   ImGui::GetIO().DisplayFramebufferScale.x = backbuffer_scale;
   ImGui::GetIO().DisplayFramebufferScale.y = backbuffer_scale;
   ImGui::GetIO().FontGlobalScale = backbuffer_scale;
+  ImGui::GetStyle() = {};
```

Comment:
This doesn't look right. What are you trying to do here exactly?

Reply:
ScaleAllSizes scales the existing sizes.  So if you call it twice with 2, you get 4x.  Call it three times, and you get 8x.

Should I add a comment, or would it make more sense to separate the change into a separate commit with a message describing the issue there?

Reply:
Definitely feels comment worthy to me

Reply:
Done